### PR TITLE
Updating the wording of the certificate view page

### PR DESCRIPTION
**Pull request by OmarIthawi**
Task:

 - [The Text under "Course Certificate" page should be edited](https://app.asana.com/0/96288420553298/65966205208542)

### CODE REVIEW

**Comment by ahmedaljazzar** [comment]
LGTM 👍

**Comment by ahmedaljazzar** [comment]
@halawa are you interested in taking a look into this?

File: lms/templates/edraak_certificates/view.html [hunk 1]
```diff
@@ -28,13 +28,20 @@
...
-                          link_end='</a>',
+                        ${_("Please note that the certificate will be issued with your full name on it as provided "
+                            "on your {link_start}account settings{link_end}.").format(
+                              link_start='<a target="_blank" href="{}">'.format(reverse('account_settings')),
+                              link_end='</a>',
+                        )}
+                    </p>
+
+                    <p>
+                        ${_("Please double check your name. We prefer "
```
Comment (ahmedaljazzar) [inline lms/templates/edraak_certificates/view.html]: I can see two prefers in **we** *prefer prefer* **that**

**Comment by ahmedaljazzar** [comment]
@OmarIthawi, I can't find the changed text in the task, do we supposed to get a one from our minds? The only thing I understood from the task is that we need to break the line and to change few words in the translation, is this true?

File: lms/templates/edraak_certificates/view.html (hunk 1 above)
Comment (OmarIthawi) [inline lms/templates/edraak_certificates/view.html]: Done. Perfect catch 👍

**Pull request merged**